src/utils/api.js

```diff
@@ -1,16 +1,18 @@
 import axios from 'axios';
 
+const baseURL = 'https://news-app-2.onrender.com/api'
+
 export const getArticles = ()=>{
     return axios
-    .get(`https://news-app-2.onrender.com/api/articles`)
+    .get(`${baseURL}/articles`)
```

Comment:
You could use axios.create to make this even easier and not require stringliterals